app.py

```diff
@@ -1,11 +1,30 @@
-from flask import Flask, jsonify, render_template
+from flask import Flask, jsonify, render_template, request
 from tablero.Dashboard import get_dashboard
 app = Flask(__name__)
 
 @app.route('/')
 def index():
     return render_template("index.html", tablero=get_dashboard())
 
+@app.route('/api/v1/record', methods=['POST'])
+def add_new_record():
+    datafile = 'data/testmake.log.csv'
+    repo = request.args['repo']
+    objetivo = request.args['objetivo']
+    revision = request.args['revision']
+    id = request.args['id']
+    phony = request.args['phony']
+    analista = request.args['analista']
+    maquina = request.args['maquina']
+    timestamp = request.args['timestamp']
+    es_make_exitoso = request.args['es_make_exitoso']
+    es_phony = request.args['es_phony']
+    existe_objetivo = request.args['existe_objetivo']
+    with open(datafile, 'a') as archivo:
+        archivo.write(f"{repo},{objetivo},{revision},{id},{phony},{analista},{maquina},{timestamp},{es_make_exitoso},{es_phony},{existe_objetivo}\n")
+    archivo.close()
+    return jsonify(request.args)
```

Comment:
Estaba leyendo este _thread_ https://stackoverflow.com/questions/19199872/best-practice-for-restful-post-response, parece que podrías regresar el registro completo que guardas en el csv, incluyendo id, timestamp ...

Reply:
Sí, eso es lo que estoy regresando

Reply:
![image](https://user-images.githubusercontent.com/3461018/61834486-fbee4000-ae2c-11e9-8dc5-90ac4b4d23b8.png)